Assets/_nathan/materials/outdoors/Tree_trunk.mat

```diff
@@ -0,0 +1,77 @@
+%YAML 1.1
```

Comment:
We already have a bunch of prototype materials for trees, water, grass etc. in `Characters/Materials` we should only have one set of prototype materials. We can merge this in an unify in the next step (I have a tool that will help) but moving forwards lets put all prototype materials into the common folder to avoid duplication.